src/Fluent/Logger/FileLogger.php

```diff
@@ -102,7 +102,7 @@ protected function postImpl(Entity $entity)
             if (!flock($this->fp, LOCK_EX)) {
                 throw new \Exception('could not obtain LOCK_EX');
             }
-            fseek($this->fp, -1, SEEK_END);
+            fseek($this->fp, 0, SEEK_END);
```

Comment:
0 looks right.
according to the manual,  the number means a position **before** the end-of-file. so zero means the very end.